kobolds/biology.md

```diff
@@ -64,8 +112,8 @@ the spine and into the tale. These are generally less than a half inch long and
 while they may look fearsome, they are not very sharp but might be useful for a
 good scratch.
 
-Roughly every 15 months, kobolds will molt. The process takes between 10 and 30
-days. Like lizards, molting happens in stages, which the outer layer of skin
+Roughly every 15 months, a kobold will molt. The process takes between 10 and 30
```

Comment:
It would be weird if this were not tied to an environmental cycle length. For example, if this planet has a 12 month year, why would Kobolds evolve to be constantly molting in different seasons rather than being predictable?

Reply:
Hmm. Hmm hmm. I shall do research.

Reply:
Looks like real lizards molt in highly variable ways. Some will shed every few weeks, some every few months (4? 6?). The Komodo Dragon apparently molts every 6 months and it _takes_ 6 months. Sounds exhausting.

The other thing, though, is that molting is more frequent when younger because they're growing faster. In fact, it looks like growth is the key driver of molting. So I probably need to characterize this more broadly anyway. Maybe I'll make it "roughly every year" for a typical adult and then mention that kids do it a lot more and elders somewhat less.